docs/RELEASE_NOTES.md

```diff
@@ -27,17 +27,34 @@
 7. Add Migration guide section which specifies explicitly if there are breaking changes and how to tackle them.
 
 -->
+
 <!--
-### 1.6.1 -  Oct 16, 2019
-[Commits](https://github.com/commercetools/commercetools-sync-java/compare/1.6.0...1.1.0) |
+### 1.7.0 -  Oct 31, 2019
+[Commits](https://github.com/commercetools/commercetools-sync-java/compare/1.6.1...1.7.0) |
+[Javadoc](https://commercetools.github.io/commercetools-sync-java/v/1.7.0/) | 
+[Jar](https://bintray.com/commercetools/maven/commercetools-sync-java/1.7.0)
+-->
+
+
+### 1.6.1 -  Oct 17, 2019
+[Commits](https://github.com/commercetools/commercetools-sync-java/compare/1.6.0...1.6.1) |
 [Javadoc](https://commercetools.github.io/commercetools-sync-java/v/1.6.1/) | 
 [Jar](https://bintray.com/commercetools/maven/commercetools-sync-java/1.6.1)
 
-- 🐞 **Bug Fixes** (1)
+- 🐞 **Bug Fixes** (3)
     - **Commons** - Fixed a bug in the `CtpQueryUtils` which was overwriting the query input query for every page after
```

Comment:
```suggestion
    - **Commons** - Fixed a bug in the `CtpQueryUtils` which was overwriting the  input query for every page after
```

Reply:
Why? It looks like the indentation is consistent with the previous release notes?

Reply:
I think there is a "query" to much ...query input query